### DISCUSSION TITLE
Red triangle at the monitor adapter

### DISCUSSION THREAD

**Discussion by Falconcrest** [nzymedefense/nzyme#540] [Questions] [answered]
I use 10 USB WiFi adapters on a RPi4 4Gb, 8 for monitoring (2,4GHz / 5GHz) and 2 for traps. 
I noticed one thing, that I cannot fully explain. 
If I choose the distribution of the channels to the 8 adapters for the monitoring "unfavorable", that is, an adapter uses 4 channels on which nobody is transmitting, then after a while a red warning triangle appears by this adapter in nzyme. 
In my experience, this effect no longer resolves itself. 
If I make a "more skillful" selection of channels, that is, if there is at least 1 highly frequented channel among them, then everything is fine.
Is there a reason for this behavior?

**Comment by lennartkoopmann** [comment, marked as answer]
Yes, this means that nzyme "thinks" the probe/adapter has issues because it has not received any frames. This is unfortunate and will be fixed in future versions. Please ignore it for now.

**Reply by Falconcrest** [reply]
Thanks for the answer, I will ignore it. 
Sorry, of course I don't mean your answer! ;-)

**Reply by lennartkoopmann** [reply]
I'm interested in how you managed to get 10 adapters connected to an RPI without power issues. I guess you used a powered hub? Write it up if you want! :)

**Reply by Falconcrest** [reply]
Ok, here comes my configuration:
As already mentioned, there is an RPi4 in use. I use DietPi as the operating system. The system runs on an SSD connected by an adapter, so a USB port is used. A activ 7-way USB hub each for 4 Ralink WiFi adapters is connected to USB ports 2 and 3. A activ 4-way USB hub for 2 Ralink WiFi adapters is connected to USB port 4. I always left one port free in the USB hub, i.e. 1-3-5-7, 1-3-5-7 and 1-3.
The two 7-way USB hubs supply the eight WiFi adapters for PROBES and the 4-way USB hub the two WiFi adapters for TRAPS.
It has been working perfectly so far and can even be expanded thanks to the free ports in the hubs.

There is still a small downer: the problem with the callbacks that I described here has not yet been solved.
I just can't get it to work.
Sorry for my bad English (Google Translate). :-(

**Reply by lennartkoopmann** [reply]
Thank you!

**Reply by lennartkoopmann** [reply]
Quick update: I'll introduce a variable to control for how long a probe/monitor can be idle without raising the alert you are seeing. Ticket is #559

**Reply by lennartkoopmann** [reply]
Done and will be included in v1.2.0. Thanks again.
